settings.gradle

```diff
@@ -11,6 +11,7 @@ pluginManagement {
                 }
             }
         }
+        mavenLocal()
```

Comment:
I don't think we want this in general

Reply:
Removed, good catch this was for local testing